From bbf184e3bcfac8ffaae796f4038bc7bf495ae7b3 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Tue, 2 Nov 2021 06:54:26 -0700
Subject: Automator: update proxy@master in istio/istio@master (#35820)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 50e16eca35..73b9423639 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "12165d1e8be6f913b354692a519129cbc78c9eed"
+    "lastStableSHA": "1829f56082533f8ab8ddcc8a33f74b7f4fdbfb08"
   }
 ]
-- 
2.35.3

