From 842d9e01c79a48892055976c55ef243bfaa5a12c Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Wed, 3 Nov 2021 15:10:20 -0700
Subject: Automator: update proxy@master in istio/istio@master (#35890)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 48e82531aa..ee69d3ab3b 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "86e400adeed007c008e43625704f50ff4d497106"
+    "lastStableSHA": "9e638d2aebbedb03204df21a5b13fa4572f91eaa"
   }
 ]
-- 
2.35.3

