From 7159f27d021a432ec82190fe6099bcd5d8da4ebe Mon Sep 17 00:00:00 2001
From: Eric Van Norman <ericvn@us.ibm.com>
Date: Thu, 4 Nov 2021 21:04:28 -0500
Subject: Update to fix hadolint warning (#35889)

---
 samples/bookinfo/src/productpage/Dockerfile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/samples/bookinfo/src/productpage/Dockerfile b/samples/bookinfo/src/productpage/Dockerfile
index 4f851116f3..a6c97402c3 100644
--- a/samples/bookinfo/src/productpage/Dockerfile
+++ b/samples/bookinfo/src/productpage/Dockerfile
@@ -14,6 +14,7 @@
 
 FROM python:3.7.7-slim
 
+WORKDIR /
 COPY requirements.txt ./
 RUN pip install --no-cache-dir -r requirements.txt
 
-- 
2.35.3

