From babb296f061f80d297e8a4da18add60a12ca3525 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Tue, 9 Nov 2021 07:18:55 -0800
Subject: Automator: update proxy@master in istio/istio@master (#35956)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 94a768beac..12d204fa6c 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "ac5a0a019388a52cbfef86d3797a5f7f5996a09f"
+    "lastStableSHA": "5aa50f0625de20ce26eaf6048b55220760e526ed"
   }
 ]
-- 
2.35.3

