From 0da7ad7de61249b5a26ebf1fea15845a47f20f92 Mon Sep 17 00:00:00 2001
From: Zhonghu Xu <xuzhonghu@huawei.com>
Date: Thu, 11 Nov 2021 22:13:02 +0800
Subject: make log more readable (#36020)

---
 pilot/pkg/xds/discovery.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pilot/pkg/xds/discovery.go b/pilot/pkg/xds/discovery.go
index 2144e96b7a..6ca7d4ab95 100644
--- a/pilot/pkg/xds/discovery.go
+++ b/pilot/pkg/xds/discovery.go
@@ -552,7 +552,7 @@ func (s *DiscoveryServer) initPushContext(req *model.PushRequest, oldPushContext
 	push.PushVersion = version
 	push.JwtKeyResolver = s.JwtKeyResolver
 	if err := push.InitContext(s.Env, oldPushContext, req); err != nil {
-		log.Errorf("XDS: Failed to update services: %v", err)
+		log.Errorf("XDS: failed to init push context: %v", err)
 		// We can't push if we can't read the data - stick with previous version.
 		pushContextErrors.Increment()
 		return nil, err
-- 
2.35.3

