From 46e1f19f6ed1290b6e0bfb8c6976696bd6401aa2 Mon Sep 17 00:00:00 2001
From: John Howard <howardjohn@google.com>
Date: Thu, 11 Nov 2021 07:02:34 -0800
Subject: Start using metallb for IPv6 (#36010)

---
 prow/integ-suite-kind.sh | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/prow/integ-suite-kind.sh b/prow/integ-suite-kind.sh
index 3533bb0222..316d6e87a6 100755
--- a/prow/integ-suite-kind.sh
+++ b/prow/integ-suite-kind.sh
@@ -104,12 +104,8 @@ done
 # Default IP family of the cluster is IPv4
 export IP_FAMILY="${IP_FAMILY:-ipv4}"
 
-# KinD will not have a LoadBalancer, so we need to disable it
-export TEST_ENV=kind
-# LoadBalancer in Kind is supported using metallb if not ipv6.
-if [ "${IP_FAMILY}" != "ipv6" ]; then
-  export TEST_ENV=kind-metallb
-fi
+# LoadBalancer in Kind is supported using metallb
+export TEST_ENV=kind-metallb
 
 # See https://kind.sigs.k8s.io/docs/user/quick-start/#loading-an-image-into-your-cluster
 export PULL_POLICY=IfNotPresent
-- 
2.35.3

