From d1bcf86bb4d73d544cd0ec6adc9fc8e2db3bcc02 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Fri, 12 Nov 2021 12:06:19 -0800
Subject: Automator: update proxy@master in istio/istio@master (#36060)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 8d7f338090..d3585367b0 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "c536aadbd86de529bba1ffc76687f5719e65353d"
+    "lastStableSHA": "50167b73dd059662a769858d7e6604e3acd37092"
   }
 ]
-- 
2.35.3

