From 1c978bdb9cfb16525c8ae215243a146eed73f6b8 Mon Sep 17 00:00:00 2001
From: Pengyuan Bian <bianpengyuan@google.com>
Date: Mon, 15 Nov 2021 12:10:45 -0800
Subject: Add silence usage to CNI command. (#36079)

---
 cni/pkg/cmd/root.go | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/cni/pkg/cmd/root.go b/cni/pkg/cmd/root.go
index 7c8e9aed8c..721c5de142 100644
--- a/cni/pkg/cmd/root.go
+++ b/cni/pkg/cmd/root.go
@@ -45,8 +45,9 @@
 )
 
 var rootCmd = &cobra.Command{
-	Use:   "install-cni",
-	Short: "Install and configure Istio CNI plugin on a node, detect and repair pod which is broken by race condition.",
+	Use:          "install-cni",
+	Short:        "Install and configure Istio CNI plugin on a node, detect and repair pod which is broken by race condition.",
+	SilenceUsage: true,
 	RunE: func(cmd *cobra.Command, args []string) (err error) {
 		if err := log.Configure(logOptions); err != nil {
 			log.Errorf("Failed to configure log %v", err)
@@ -56,7 +57,6 @@
 		// Start controlz server
 		_, _ = ctrlz.Run(ctrlzOptions, nil)
 
-		// TODO(bianpengyuan) add log scope for install & repair.
 		var cfg *config.Config
 		if cfg, err = constructConfig(); err != nil {
 			return
-- 
2.35.3

