From 208817432e0f2efd7174af2a35a0ecdb0a78f71f Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Fri, 19 Nov 2021 03:53:04 -0800
Subject: Automator: update proxy@master in istio/istio@master (#36154)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index af718c7f57..65ab1cc10b 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "08d901c958851300601eb56b9232ac117fdfe7d6"
+    "lastStableSHA": "70a030877f9514da977fcf892506a925038ab74a"
   }
 ]
-- 
2.35.3

