From 983f785d8bd524b12aa4e7506d4a05128cb5fc8a Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Fri, 19 Nov 2021 17:14:45 -0800
Subject: Automator: update proxy@master in istio/istio@master (#36176)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 65ab1cc10b..97f13279c4 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "70a030877f9514da977fcf892506a925038ab74a"
+    "lastStableSHA": "fa1b725bdddb18222fa0d676147ba46f49fbaed3"
   }
 ]
-- 
2.35.3

