From 0d3afbbfe04fc4f2cace10a7878b3025e6d93655 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Wed, 24 Nov 2021 17:18:16 -0800
Subject: Automator: update proxy@master in istio/istio@master (#36252)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 13f1fc3bb5..6e20b969f4 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "afcf498a60787a715a4cbe3c15f84b48d9b39050"
+    "lastStableSHA": "3d66d6e1b614e863e8028da1d6eda63a2c6f264f"
   }
 ]
-- 
2.35.3

