From 0d6d80d594b1b7048b6d92afe9bc0c75db3fdb34 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Thu, 25 Nov 2021 17:25:59 -0800
Subject: Automator: update proxy@master in istio/istio@master (#36266)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 6e20b969f4..e410a3245d 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "3d66d6e1b614e863e8028da1d6eda63a2c6f264f"
+    "lastStableSHA": "8043fff1683952834b511d7326c49e1251c305fa"
   }
 ]
-- 
2.35.3

