From 57053f58ecdda6f1b63e4fc43b2e43331709f3c8 Mon Sep 17 00:00:00 2001
From: Xiaopeng Han <hanxiaop8@gmail.com>
Date: Wed, 1 Dec 2021 10:09:53 +0800
Subject: fix message being omitted (#36253)

---
 istioctl/cmd/precheck.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istioctl/cmd/precheck.go b/istioctl/cmd/precheck.go
index 588ff454f8..8e49b81a17 100644
--- a/istioctl/cmd/precheck.go
+++ b/istioctl/cmd/precheck.go
@@ -140,7 +140,7 @@ func checkControlPlane(cli kube.ExtendedClient) (diag.Messages, error) {
 		return nil, err
 	}
 	if result.Messages != nil {
-		msgs = result.Messages
+		msgs = append(msgs, result.Messages...)
 	}
 
 	return msgs, nil
-- 
2.35.3

