From 8835b8da622a62bb55e47a46b2b2a26c1be53d83 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Fri, 3 Dec 2021 20:08:49 -0800
Subject: Automator: update proxy@master in istio/istio@master (#36374)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index f78cbdd6dd..6c3c29fa7b 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "7d054b44023a3967b33ab76d9acf4f873081cb5d"
+    "lastStableSHA": "24ab07ecba821389e3dd3b7894327bd657abd52a"
   }
 ]
-- 
2.35.3

