From 5dd861d4ed1efef56c74bb6ecf97fca60b691a10 Mon Sep 17 00:00:00 2001
From: John Howard <howardjohn@google.com>
Date: Tue, 7 Dec 2021 12:06:04 -0800
Subject: Fix log spam for Gateway status controller (#36165)

---
 pilot/pkg/status/resourcelock.go | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/pilot/pkg/status/resourcelock.go b/pilot/pkg/status/resourcelock.go
index 86683506ad..20dedc0a58 100644
--- a/pilot/pkg/status/resourcelock.go
+++ b/pilot/pkg/status/resourcelock.go
@@ -204,11 +204,9 @@ func (wp *WorkerPool) maybeAddWorker() {
 			if cfg != nil {
 				// Check that generation matches
 				if strconv.FormatInt(cfg.Generation, 10) == target.Generation {
-					var x GenerationProvider
 					x, err := GetOGProvider(cfg.Status)
-					if err != nil {
-						scope.Warnf("status has no observed generation, overwriting: %s", err)
-					} else {
+					if err == nil {
+						// Not all controllers user generation, so we can ignore errors
 						x.SetObservedGeneration(cfg.Generation)
 					}
 					for c, i := range perControllerWork {
-- 
2.35.3

