From eeaeb2b0a83705cb4e89a4dc06cde1827d288fa9 Mon Sep 17 00:00:00 2001
From: John Howard <howardjohn@google.com>
Date: Wed, 8 Dec 2021 14:07:52 -0800
Subject: Make fake testing mirror more robust (#36441)

---
 pkg/kube/fakemirror.go | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/pkg/kube/fakemirror.go b/pkg/kube/fakemirror.go
index ed18345223..2da40fbebb 100644
--- a/pkg/kube/fakemirror.go
+++ b/pkg/kube/fakemirror.go
@@ -53,6 +53,10 @@ func mirrorTo(q queue.Instance, createClient interface{}, convert convertFn) *un
 func (c *untypedMirror) OnAdd(obj interface{}) {
 	meta := obj.(metav1.Object)
 	res := c.convertRes(obj)
+	if res == nil {
+		log.Warnf("failed to mirror resource %v/%v", meta.GetName(), meta.GetName())
+		return
+	}
 	log.Debugf("Mirroring ADD %s/%s", meta.GetName(), meta.GetName())
 	c.queue.Push(func() error {
 		return c.Create(meta.GetNamespace(), res)
@@ -62,6 +66,10 @@ func (c *untypedMirror) OnAdd(obj interface{}) {
 func (c *untypedMirror) OnUpdate(_, obj interface{}) {
 	meta := obj.(metav1.Object)
 	res := c.convertRes(obj)
+	if res == nil {
+		log.Warnf("failed to mirror resource %v/%v", meta.GetName(), meta.GetName())
+		return
+	}
 	res.SetResourceVersion("")
 	log.Debugf("Mirroring UPDATE %s/%s", meta.GetName(), meta.GetName())
 	c.queue.Push(func() error {
-- 
2.35.3

