From e5e8721d73efbaedb64c138bf3dad3a58a57304d Mon Sep 17 00:00:00 2001
From: John Howard <howardjohn@google.com>
Date: Wed, 8 Dec 2021 14:46:06 -0800
Subject: Move to v1.23.0 for main tests (#36443)

---
 prow/integ-suite-kind.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/prow/integ-suite-kind.sh b/prow/integ-suite-kind.sh
index 316d6e87a6..1a5b2ad471 100755
--- a/prow/integ-suite-kind.sh
+++ b/prow/integ-suite-kind.sh
@@ -37,7 +37,7 @@ setup_and_export_git_sha
 source "${ROOT}/common/scripts/kind_provisioner.sh"
 
 TOPOLOGY=SINGLE_CLUSTER
-NODE_IMAGE="gcr.io/istio-testing/kind-node:v1.22.2"
+NODE_IMAGE="gcr.io/istio-testing/kind-node:v1.23.0"
 KIND_CONFIG=""
 CLUSTER_TOPOLOGY_CONFIG_FILE="${ROOT}/prow/config/topology/multicluster.json"
 
-- 
2.35.3

