From 1a53bafc14fee84cc7c6c49fff5c1a0d82740297 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Sun, 12 Dec 2021 14:56:21 -0800
Subject: Automator: update proxy@master in istio/istio@master (#36481)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 4612f21845..9bdcfbc46e 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "2a2975bcd63755c0a7114093d5c323bfadf08ff8"
+    "lastStableSHA": "63485533cce5bac350e3760ce7b78d7e2534caa2"
   }
 ]
-- 
2.35.3

