From 219710c67961153623f7f00c07c72a98af8cea3a Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Tue, 14 Dec 2021 17:33:09 -0800
Subject: Automator: update proxy@master in istio/istio@master (#36515)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index f1f6581d35..710f45f20e 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "bc8643bbe46d4c44dfd091dc9144f7bdb9f4c0a4"
+    "lastStableSHA": "1a1147e385d98cc1c9cb18a50cf5bcc5eb354497"
   }
 ]
-- 
2.35.3

