From 873ba55abcd95cd0b438514021f6bf7e9d498579 Mon Sep 17 00:00:00 2001
From: Pengyuan Bian <bianpengyuan@google.com>
Date: Sat, 18 Dec 2021 01:51:51 -0800
Subject: remove a debug log. (#36559)

---
 pkg/webhooks/webhookpatch.go | 1 -
 1 file changed, 1 deletion(-)

diff --git a/pkg/webhooks/webhookpatch.go b/pkg/webhooks/webhookpatch.go
index 96df25f5c1..8eb81ca2c5 100644
--- a/pkg/webhooks/webhookpatch.go
+++ b/pkg/webhooks/webhookpatch.go
@@ -97,7 +97,6 @@ func (w *WebhookCertPatcher) HasSynced() bool {
 
 // webhookPatchTask takes the result of patchMutatingWebhookConfig and modifies the result for use in task queue
 func (w *WebhookCertPatcher) webhookPatchTask(o types.NamespacedName) error {
-	log.Errorf("howardjohn: patch %v", o)
 	reportWebhookPatchAttempts(o.Name)
 	err := w.patchMutatingWebhookConfig(
 		w.client.AdmissionregistrationV1().MutatingWebhookConfigurations(),
-- 
2.35.3

