From 7ab13d9a13e338bec31c00fe36f95bdbddc1f640 Mon Sep 17 00:00:00 2001
From: Rama Chavali <rama.rao@salesforce.com>
Date: Thu, 23 Dec 2021 14:17:58 +0530
Subject: add additional logging for termination drain logic (#36618)

Signed-off-by: Rama Chavali <rama.rao@salesforce.com>
---
 pkg/envoy/agent.go | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/pkg/envoy/agent.go b/pkg/envoy/agent.go
index 8edfbb09fd..b5130cebb7 100644
--- a/pkg/envoy/agent.go
+++ b/pkg/envoy/agent.go
@@ -149,11 +149,13 @@ func (a *Agent) terminate() {
 		log.Infof("Checking for active connections...")
 		ticker := time.NewTicker(activeConnectionCheckDelay)
 		for range ticker.C {
-			if a.activeProxyConnections() == 0 {
+			ac := a.activeProxyConnections()
+			if ac == 0 {
 				log.Info("There are no more active connections. terminating proxy...")
 				a.abortCh <- errAbort
 				return
 			}
+			log.Infof("There are still %d active connections", ac)
 		}
 	} else {
 		log.Infof("Graceful termination period is %v, starting...", a.terminationDrainDuration)
@@ -199,6 +201,9 @@ func (a *Agent) activeProxyConnections() int {
 		}
 		activeConnections += int(val)
 	}
+	if activeConnections > 0 {
+		log.Debugf("Active connections stats: %s", stats.String())
+	}
 	return activeConnections
 }
 
-- 
2.35.3

