From 0a9fd488f111f48bb5084862900fcb0f5b0a8089 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Fri, 24 Dec 2021 18:50:47 -0800
Subject: Automator: update proxy@master in istio/istio@master (#36632)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 58d03719da..e1f88bcaba 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "57923d1c967e92fb74d99756ec9e4212fcea20dd"
+    "lastStableSHA": "39ebaaf2f85c48dee3256b0f8ab2101286676a13"
   }
 ]
-- 
2.35.3

