From a6454abd17ac4c4c2a6929077f619947caf7c6d7 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Fri, 24 Dec 2021 23:17:21 -0800
Subject: Automator: update proxy@master in istio/istio@master (#36635)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index e1f88bcaba..9570efa11d 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "39ebaaf2f85c48dee3256b0f8ab2101286676a13"
+    "lastStableSHA": "bcc7b9046521dcc62f00befb5fedb9b090d6c873"
   }
 ]
-- 
2.35.3

