From 7b6846c8f423b4b9db107122b158951fceb0031c Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Tue, 28 Dec 2021 17:30:57 -0800
Subject: Automator: update proxy@master in istio/istio@master (#36654)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 9570efa11d..f2a17754b5 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "bcc7b9046521dcc62f00befb5fedb9b090d6c873"
+    "lastStableSHA": "58bb5bd4bb559c31f488c3ea489b76c634d2e427"
   }
 ]
-- 
2.35.3

