From a07fb3c80e694adeea5bd9bb27bc6a242ff4b2c3 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Thu, 30 Dec 2021 05:16:17 -0800
Subject: Automator: update proxy@master in istio/istio@master (#36662)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index f2a17754b5..ea0577b594 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "58bb5bd4bb559c31f488c3ea489b76c634d2e427"
+    "lastStableSHA": "e0dfbac609dbcafd1e9024a10083276508d92b67"
   }
 ]
-- 
2.35.3

