From 117570502cdd0d9c2511ddc8d79529caf56dc581 Mon Sep 17 00:00:00 2001
From: Morven Cao <lcao@redhat.com>
Date: Sun, 2 Jan 2022 10:29:16 +0800
Subject: remove debug log. (#36668)

---
 pkg/webhooks/webhookpatch.go | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/pkg/webhooks/webhookpatch.go b/pkg/webhooks/webhookpatch.go
index 8eb81ca2c5..e36923ae58 100644
--- a/pkg/webhooks/webhookpatch.go
+++ b/pkg/webhooks/webhookpatch.go
@@ -17,7 +17,6 @@
 import (
 	"bytes"
 	"context"
-	"encoding/json"
 	"errors"
 	"fmt"
 	"strings"
@@ -128,8 +127,6 @@ func (w *WebhookCertPatcher) patchMutatingWebhookConfig(
 	// prevents a race condition between multiple istiods when the revision is changed or modified
 	v, ok := config.Labels[label.IoIstioRev.Name]
 	if v != w.revision || !ok {
-		debug, _ := json.MarshalIndent(config, "howardjohn", "  ")
-		log.Errorf("howardjohn: %s", debug)
 		reportWebhookPatchFailure(webhookConfigName, reasonWrongRevision)
 		return errWrongRevision
 	}
-- 
2.35.3

