From 54c6fdf97b764a983cce95bb1bb55fa5a5916099 Mon Sep 17 00:00:00 2001
From: Xiaopeng Han <hanxiaop8@outlook.com>
Date: Mon, 3 Jan 2022 19:48:16 +0800
Subject: add info log (#36680)

---
 operator/pkg/helmreconciler/reconciler.go | 1 +
 1 file changed, 1 insertion(+)

diff --git a/operator/pkg/helmreconciler/reconciler.go b/operator/pkg/helmreconciler/reconciler.go
index 30416077e8..b925585115 100644
--- a/operator/pkg/helmreconciler/reconciler.go
+++ b/operator/pkg/helmreconciler/reconciler.go
@@ -517,6 +517,7 @@ func (h *HelmReconciler) reportPrunedObjectKind() {
 // CreateNamespace creates a namespace using the given k8s interface.
 func CreateNamespace(cs kubernetes.Interface, namespace string, network string, dryRun bool) error {
 	if dryRun {
+		scope.Infof("Not applying Namespace %s because of dry run.", namespace)
 		return nil
 	}
 	if namespace == "" {
-- 
2.35.3

