From 6652cbab960a2175b0fe273d9dc3687bdd14c964 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Tue, 4 Jan 2022 19:01:50 -0800
Subject: Automator: update proxy@master in istio/istio@master (#36704)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 0918b17359..3cbd894e93 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "2ebad2f7c5e712748ecc4171537b77a9b370c94f"
+    "lastStableSHA": "3002cc917b3252faae1f2a163bab1ccc32352a5f"
   }
 ]
-- 
2.35.3

