From 8af659a938fc50383a3400d0f73c5f6a149053db Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Wed, 5 Jan 2022 16:49:22 -0800
Subject: Automator: update proxy@master in istio/istio@master (#36720)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 3cbd894e93..1968c5abab 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "3002cc917b3252faae1f2a163bab1ccc32352a5f"
+    "lastStableSHA": "9dd2fc92ad2ace578535f0e1429342bee184a136"
   }
 ]
-- 
2.35.3

