From 163876846058a96f84e671302ca6e7430846c020 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Thu, 6 Jan 2022 15:00:36 -0800
Subject: Automator: update proxy@master in istio/istio@master (#36739)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 1968c5abab..16da3122a2 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "9dd2fc92ad2ace578535f0e1429342bee184a136"
+    "lastStableSHA": "4f4173257df021356c10bad23bdb8946a4dd9ccc"
   }
 ]
-- 
2.35.3

