From 59e558317e7d1eea499608cad5ffd5df99eb95b4 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Sat, 8 Jan 2022 17:03:06 -0800
Subject: Automator: update proxy@master in istio/istio@master (#36761)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 6f606b680c..42eaf6db07 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "52a86aa341d9b5d4d8b5127277f81bbd0c4db16a"
+    "lastStableSHA": "4de700407b62c6e2b5ad04aa36924d56ac1c0bba"
   }
 ]
-- 
2.35.3

