From a0e25ff34bc5cf11ad92260e90acb59358caa12a Mon Sep 17 00:00:00 2001
From: John Howard <howardjohn@google.com>
Date: Mon, 10 Jan 2022 11:16:50 -0800
Subject: Fix empty variable check (#36774)

---
 tools/docker | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/docker b/tools/docker
index 072170c800..2033d9ebd1 100755
--- a/tools/docker
+++ b/tools/docker
@@ -21,7 +21,7 @@ ROOT=$(dirname "$WD")
 set -eu
 
 # LOCAL_OUT is set when runing in a Makefile and set to current architecture (not host)
-if [[ ! -z "${LOCAL_OUT}" ]]; then
+if [[ ! -z "${LOCAL_OUT+.}" ]]; then
   OS_ARCH=${LOCAL_OUT##*/}
   LOCAL_TARGET_OS=${OS_ARCH%_*}
   LOCAL_TARGET_ARCH=${OS_ARCH##*_}
-- 
2.35.3

