From c012d0674b25ee53c966127150400ff0362992f7 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Tue, 11 Jan 2022 10:40:27 -0800
Subject: Automator: update proxy@master in istio/istio@master (#36786)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 38c647d27a..46b72e2899 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "ba89d2ee85afffa179a9cc1c34abc330236c4147"
+    "lastStableSHA": "98c9ae552e6300c93ba1215c545604f1c257e96e"
   }
 ]
-- 
2.35.3

