From 5d0010a76085326f60eb23cd25d5c9c3888d4514 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Wed, 19 Jan 2022 18:30:18 -0800
Subject: Automator: update proxy@master in istio/istio@master (#36902)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 8bc2946523..9a03491fc3 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "0a4620bdfa4ccae5b442f6afae38be9869ff82ef"
+    "lastStableSHA": "f7745f5fb2207ef2177207922c48ad074266801d"
   }
 ]
-- 
2.35.3

