From 93ad73b05bb4ba31df36ce84dd1e1ab94459d219 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Fri, 21 Jan 2022 07:27:55 -0800
Subject: Automator: update proxy@master in istio/istio@master (#36923)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 9a03491fc3..7fd719db1a 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "f7745f5fb2207ef2177207922c48ad074266801d"
+    "lastStableSHA": "1bfd5d9a0fa9ce1686989a71897165286e552bf1"
   }
 ]
-- 
2.35.3

