From 09b7ae45c489290c3434fccf5d2e6a7fa6162a26 Mon Sep 17 00:00:00 2001
From: John Howard <howardjohn@google.com>
Date: Thu, 27 Jan 2022 00:54:59 -0800
Subject: fix format of error in istioctl (#37017)

---
 operator/pkg/validate/common.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/operator/pkg/validate/common.go b/operator/pkg/validate/common.go
index 71738508a0..18297b9d27 100644
--- a/operator/pkg/validate/common.go
+++ b/operator/pkg/validate/common.go
@@ -93,7 +93,7 @@
 func validateWithRegex(path util.Path, val interface{}, r *regexp.Regexp) (errs util.Errors) {
 	valStr := fmt.Sprint(val)
 	if len(r.FindString(valStr)) != len(valStr) {
-		errs = util.AppendErr(errs, fmt.Errorf("invalid value %s: %s", path, val))
+		errs = util.AppendErr(errs, fmt.Errorf("invalid value %s: %v", path, val))
 		printError(errs.ToError())
 	}
 	return errs
-- 
2.35.3

