From 8095051f4f635ad94b493e391ffbf646b2454a13 Mon Sep 17 00:00:00 2001
From: Kebe <kebe.liu@daocloud.io>
Date: Tue, 8 Feb 2022 17:03:56 +0800
Subject: Fix ValidatingWebhookConfiguration naming in precheck (#36548)

---
 istioctl/cmd/precheck.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istioctl/cmd/precheck.go b/istioctl/cmd/precheck.go
index e042a4d302..a777a279f9 100644
--- a/istioctl/cmd/precheck.go
+++ b/istioctl/cmd/precheck.go
@@ -209,7 +209,7 @@ func checkInstallPermissions(cli kube.ExtendedClient) diag.Messages {
 		{
 			group:   "admissionregistration.k8s.io",
 			version: "v1",
-			name:    "f",
+			name:    "ValidatingWebhookConfiguration",
 		},
 	}
 	msgs := diag.Messages{}
-- 
2.35.3

