From c0c9b8cc16f82aa43b86dee2337a7cee9b69fb06 Mon Sep 17 00:00:00 2001
From: Xiaopeng Han <hanxiaop8@outlook.com>
Date: Wed, 9 Feb 2022 15:29:29 +0800
Subject: fix x authz description (#37068)

---
 istioctl/cmd/authz.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istioctl/cmd/authz.go b/istioctl/cmd/authz.go
index 866cd7f15c..d56d6b6bf4 100644
--- a/istioctl/cmd/authz.go
+++ b/istioctl/cmd/authz.go
@@ -44,7 +44,7 @@
   istioctl x authz check httpbin-88ddbcfdd-nt5jb
 
   # Check AuthorizationPolicy applied to one pod under a deployment
-  istioctl proxy-status deployment/productpage-v1
+  istioctl x authz check deployment/productpage-v1
 
   # Check AuthorizationPolicy from Envoy config dump file:
   istioctl x authz check -f httpbin_config_dump.json`,
-- 
2.35.3

