From 6d489cd53002c407d593ac47e950341f08f11a18 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Wed, 9 Feb 2022 10:18:04 -0800
Subject: Automator: update proxy@master in istio/istio@master (#37004)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 9e61d9c15a..c869a05488 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "9385dadba5de0f4018434b5552ab35e68678caed"
+    "lastStableSHA": "471a9895cfe212a4a26ac3d63ead5b8cb683a8af"
   }
 ]
-- 
2.35.3

