From 4fc57beb5433ad02b4cfdb2542016969683cd4e3 Mon Sep 17 00:00:00 2001
From: Kebe <kebe.liu@daocloud.io>
Date: Fri, 11 Feb 2022 10:29:26 +0800
Subject: Fix rootca-compare cmd validate logic error (#36551)

---
 istioctl/cmd/proxyconfig.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istioctl/cmd/proxyconfig.go b/istioctl/cmd/proxyconfig.go
index 46e401c1e0..512ad88b59 100644
--- a/istioctl/cmd/proxyconfig.go
+++ b/istioctl/cmd/proxyconfig.go
@@ -1064,7 +1064,7 @@ func rootCACompareConfigCmd() *cobra.Command {
   istioctl proxy-config rootca-compare <pod-name-1[.namespace]> <pod-name-2[.namespace]>`,
 		Aliases: []string{"rc"},
 		Args: func(cmd *cobra.Command, args []string) error {
-			if (len(args) <= 2) != (configDumpFile == "") {
+			if len(args) != 2 {
 				cmd.Println(cmd.UsageString())
 				return fmt.Errorf("rootca-compare requires 2 pods as an argument")
 			}
-- 
2.35.3

