From 7b40c137e017b3afda0559616e0ae87441337997 Mon Sep 17 00:00:00 2001
From: John Howard <howardjohn@google.com>
Date: Wed, 16 Feb 2022 12:10:22 -0800
Subject: tf: use consistent log format (#37383)

---
 tests/integration/security/main_test.go | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tests/integration/security/main_test.go b/tests/integration/security/main_test.go
index 5f60ebbbf5..63513d824d 100644
--- a/tests/integration/security/main_test.go
+++ b/tests/integration/security/main_test.go
@@ -60,7 +60,6 @@ func setupConfig(ctx resource.Context, cfg *istio.Config) {
     env: 
       PILOT_JWT_ENABLE_REMOTE_JWKS: true
 meshConfig:
-  accessLogEncoding: JSON
   accessLogFile: /dev/stdout
   defaultConfig:
     image:
-- 
2.35.3

