From 022c4908ec7c860d01d78de02dbd3fd883729fa4 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Thu, 17 Feb 2022 08:49:55 -0800
Subject: Automator: update proxy@master in istio/istio@master (#37403)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index a189519490..94212608ac 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "806ab88ff6c800309849c5200e9e6dc1f1df8bda"
+    "lastStableSHA": "26c6f9b5e8ba344d83d3b19916a79d893b0e3d4d"
   }
 ]
-- 
2.35.3

