From 786a3010e6af80546a5893e22e55c6c932f094af Mon Sep 17 00:00:00 2001
From: zirain <hejianpeng2@huawei.com>
Date: Fri, 18 Feb 2022 23:58:52 +0800
Subject: remove nit log (#37429)

---
 pilot/pkg/xds/endpoint_builder.go | 1 -
 1 file changed, 1 deletion(-)

diff --git a/pilot/pkg/xds/endpoint_builder.go b/pilot/pkg/xds/endpoint_builder.go
index 1b74de5062..4cd7efbdde 100644
--- a/pilot/pkg/xds/endpoint_builder.go
+++ b/pilot/pkg/xds/endpoint_builder.go
@@ -110,7 +110,6 @@ func NewEndpointBuilder(clusterName string, proxy *model.Proxy, push *model.Push
 	// We need this for multi-network, or for clusters meant for use with AUTO_PASSTHROUGH.
 	if features.EnableAutomTLSCheckPolicies ||
 		b.push.NetworkManager().IsMultiNetworkEnabled() || model.IsDNSSrvSubsetKey(clusterName) {
-		log.Errorf("howardjohn: dr == %v", dr == nil)
 		b.mtlsChecker = newMtlsChecker(push, port, dr)
 	}
 	return b
-- 
2.35.3

