From b21c962cb8baaf0b220d53575921b3784b96b5b8 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Thu, 24 Feb 2022 07:28:15 -0800
Subject: Automator: update proxy@master in istio/istio@master (#37511)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 733f67a93a..bf5ab42e53 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "abbc1ae46535340a6533f62495097b621f53af64"
+    "lastStableSHA": "919ec79e3c4b5d487a2eb6fb7b2a248c4b203036"
   }
 ]
-- 
2.35.3

