From 4c8178c2cdac875666f83d7d06ee1514bcf63756 Mon Sep 17 00:00:00 2001
From: John Howard <howardjohn@google.com>
Date: Fri, 25 Feb 2022 10:39:20 -0800
Subject: Trim images built for tests (#37542)

---
 prow/lib.sh | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/prow/lib.sh b/prow/lib.sh
index e3b4a873d8..0d519fba5a 100755
--- a/prow/lib.sh
+++ b/prow/lib.sh
@@ -105,11 +105,14 @@ function build_images() {
 
   # use ubuntu:bionic to test vms by default
   nonDistrolessTargets="docker.app docker.app_sidecar_ubuntu_bionic "
-  if [[ "${SELECT_TEST}" == "test.integration.pilot.kube" ]]; then
+  if [[ "${SELECT_TEST}" == "test.integration.pilot.kube" && "${JOB_TYPE}" = "postsubmit" ]]; then
+    # We run tests across all VM types only in postsubmit
     nonDistrolessTargets+="docker.app_sidecar_ubuntu_xenial docker.app_sidecar_ubuntu_focal docker.app_sidecar_ubuntu_bionic "
     nonDistrolessTargets+="docker.app_sidecar_debian_9 docker.app_sidecar_debian_10 docker.app_sidecar_centos_7 docker.app_sidecar_centos_8 "
   fi
-  targets+="docker.operator "
+  if [[ "${SELECT_TEST}" == "test.integration.operator.kube" ]]; then
+    targets+="docker.operator "
+  fi
   targets+="docker.install-cni "
   if [[ "${VARIANT:-default}" == "distroless" ]]; then
     DOCKER_BUILD_VARIANTS="distroless" DOCKER_TARGETS="${targets}" make dockerx.pushx
-- 
2.35.3

