From 42da1b98237655076830170c52ffd872806bc99e Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Sat, 26 Feb 2022 01:07:31 -0800
Subject: Automator: update proxy@master in istio/istio@master (#37564)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 548dc436bf..e30dad867d 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "3e8e6e184733c3b952f7160bded6aa448e9c0337"
+    "lastStableSHA": "d9ea31b208cb079a451893cdbdbaa82d87443088"
   }
 ]
-- 
2.35.3

