From 80c4180949ce2e0f6d25cd3d8059d6b31be8a750 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Sat, 26 Feb 2022 02:52:02 -0800
Subject: Automator: update proxy@master in istio/istio@master (#37568)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index e30dad867d..a7f7218c53 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "d9ea31b208cb079a451893cdbdbaa82d87443088"
+    "lastStableSHA": "23304f4725dda14fab0e63298a53ea162236198b"
   }
 ]
-- 
2.35.3

