From add1aee227521aa60008e599422af20a6f000ea3 Mon Sep 17 00:00:00 2001
From: Rama Chavali <rama.rao@salesforce.com>
Date: Mon, 28 Feb 2022 16:18:35 +0530
Subject: remove unused functions (#37574)

Signed-off-by: Rama Chavali <rama.rao@salesforce.com>
---
 pilot/pkg/networking/util/util.go | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/pilot/pkg/networking/util/util.go b/pilot/pkg/networking/util/util.go
index a94e88b22c..b143480ebc 100644
--- a/pilot/pkg/networking/util/util.go
+++ b/pilot/pkg/networking/util/util.go
@@ -258,18 +258,6 @@ func SortVirtualHosts(hosts []*route.VirtualHost) {
 	})
 }
 
-// IsIstioVersionGE110 checks whether the given Istio version is greater than or equals 1.10.
-func IsIstioVersionGE110(version *model.IstioVersion) bool {
-	return version == nil ||
-		version.Compare(&model.IstioVersion{Major: 1, Minor: 10, Patch: -1}) >= 0
-}
-
-// IsIstioVersionGE111 checks whether the given Istio version is greater than or equals 1.11.
-func IsIstioVersionGE111(version *model.IstioVersion) bool {
-	return version == nil ||
-		version.Compare(&model.IstioVersion{Major: 1, Minor: 11, Patch: -1}) >= 0
-}
-
 // IsIstioVersionGE112 checks whether the given Istio version is greater than or equals 1.12.
 func IsIstioVersionGE112(version *model.IstioVersion) bool {
 	return version == nil ||
-- 
2.35.3

