From b865b8f31f120ec91eea3cd9e89ce619caddca79 Mon Sep 17 00:00:00 2001
From: John Howard <howardjohn@google.com>
Date: Tue, 1 Mar 2022 07:37:37 -0800
Subject: Pilot debounce: log reason when there is no config (#37603)

---
 pilot/pkg/xds/discovery.go | 15 +++++++++++++--
 1 file changed, 13 insertions(+), 2 deletions(-)

diff --git a/pilot/pkg/xds/discovery.go b/pilot/pkg/xds/discovery.go
index 14678389d2..0c7807c6eb 100644
--- a/pilot/pkg/xds/discovery.go
+++ b/pilot/pkg/xds/discovery.go
@@ -428,8 +428,8 @@ func debounce(ch chan *model.PushRequest, stopCh <-chan struct{}, opts debounceO
 			if req != nil {
 				pushCounter++
 				if req.ConfigsUpdated == nil {
-					log.Infof("Push debounce stable[%d] %d: %v since last change, %v since last push, full=%v",
-						pushCounter, debouncedEvents,
+					log.Infof("Push debounce stable[%d] %d for reason %s: %v since last change, %v since last push, full=%v",
+						pushCounter, debouncedEvents, reasonsUpdated(req),
 						quietTime, eventDelay, req.Full)
 				} else {
 					log.Infof("Push debounce stable[%d] %d for config %s: %v since last change, %v since last push, full=%v",
@@ -496,6 +496,17 @@ func configsUpdated(req *model.PushRequest) string {
 	return configs
 }
 
+func reasonsUpdated(req *model.PushRequest) string {
+	switch len(req.Reason) {
+	case 0:
+		return "unknown"
+	case 1:
+		return string(req.Reason[0])
+	default:
+		return fmt.Sprintf("%s and %d more reasons", req.Reason[0], len(req.Reason)-1)
+	}
+}
+
 func doSendPushes(stopCh <-chan struct{}, semaphore chan struct{}, queue *PushQueue) {
 	for {
 		select {
-- 
2.35.3

