From 0290e9b5fe22ef85cae6e37a3d2b4980750c4666 Mon Sep 17 00:00:00 2001
From: John Howard <howardjohn@google.com>
Date: Tue, 1 Mar 2022 17:10:00 -0800
Subject: Restrict another hub to prevent accidents (#37591)

---
 tools/docker-builder/main.go | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/tools/docker-builder/main.go b/tools/docker-builder/main.go
index bca0bdf4da..49d042a4be 100644
--- a/tools/docker-builder/main.go
+++ b/tools/docker-builder/main.go
@@ -58,7 +58,12 @@ func main() {
 	}
 }
 
-var privilegedHubs = sets.NewSet("docker.io/istio", "istio", "gcr.io/istio-release")
+var privilegedHubs = sets.NewSet(
+	"docker.io/istio",
+	"istio",
+	"gcr.io/istio-release",
+	"gcr.io/istio-testing",
+)
 
 var rootCmd = &cobra.Command{
 	SilenceUsage: true,
-- 
2.35.3

