From 1426b3979ebaa60578b16c402aa0b6a3a47eb4fe Mon Sep 17 00:00:00 2001
From: Xiaopeng Han <hanxiaop8@outlook.com>
Date: Tue, 8 Mar 2022 04:19:01 +0800
Subject: clean up unused code (#37775)

---
 pkg/bootstrap/platform/platform.go | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/pkg/bootstrap/platform/platform.go b/pkg/bootstrap/platform/platform.go
index e0b82fd6dc..06b7caab3d 100644
--- a/pkg/bootstrap/platform/platform.go
+++ b/pkg/bootstrap/platform/platform.go
@@ -22,9 +22,6 @@
 	KubernetesServiceHost = "KUBERNETES_SERVICE_HOST"
 )
 
-// Instance is an enumeration of all supported platforms.
-type Platform int
-
 // Environment provides information for the platform on which the bootstrapping
 // is taking place.
 type Environment interface {
-- 
2.35.3

