From f0745fa13f193ecc1656ce8933331647714003d7 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Tue, 8 Mar 2022 07:40:42 -0800
Subject: Automator: update proxy@master in istio/istio@master (#37807)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 506e4a57c9..7b16daa277 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "5642c250f0a6befe890076bf36978f13d589bd23"
+    "lastStableSHA": "af96f9a2078d382cbb21b6a4efb2cd2beb2f45cd"
   }
 ]
-- 
2.35.3

