From 7071eb61ca7e8019045679959cef729a46c871ff Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Thu, 10 Mar 2022 08:23:52 -0800
Subject: Automator: update proxy@master in istio/istio@master (#37858)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 82b0e0fcc6..fa04375b88 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "249125953083f07f0d9676936aae0bbf1c1f10d8"
+    "lastStableSHA": "75393c3d79795e26ed9fc9626992720d13ac195b"
   }
 ]
-- 
2.35.3

