From 8bb103fd0d1c0d25ebf69c4d6322c78e972ba879 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Wed, 16 Mar 2022 01:42:32 -0700
Subject: Automator: update proxy@master in istio/istio@master (#37963)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 150d55ece6..d338f8e7de 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "a2348be92fb29ba4fbc7f734835a9e1b2f4913d6"
+    "lastStableSHA": "d6ad561a0d37fba5673a04f1a118eff3e7d233fd"
   }
 ]
-- 
2.35.3

