From 82e0d06d17ee199e933b89f914341ac33d20224b Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Thu, 17 Mar 2022 09:52:51 -0700
Subject: Automator: update proxy@master in istio/istio@master (#37984)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index d338f8e7de..5f2b3bc759 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "d6ad561a0d37fba5673a04f1a118eff3e7d233fd"
+    "lastStableSHA": "c64f28baa010769a0ca1bd11e7887d27fa614c16"
   }
 ]
-- 
2.35.3

