From 2b270558afa4c20b5ee8426dd8e52e4459ad2f05 Mon Sep 17 00:00:00 2001
From: Eric Van Norman <ericvn@us.ibm.com>
Date: Mon, 21 Mar 2022 18:02:24 -0500
Subject: Fix BUILD_WITH_CONTAINER=0 make docker.* (#38045)

---
 tools/istio-docker.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/istio-docker.mk b/tools/istio-docker.mk
index 44541ecdb6..85cec4a497 100644
--- a/tools/istio-docker.mk
+++ b/tools/istio-docker.mk
@@ -57,7 +57,7 @@ dockerx: docker
 # Support individual images like `dockerx.pilot`
 
 # Docker commands defines some convenience targets
-define DOCKER_COMMANDS =
+define DOCKER_COMMANDS
 # Build individual docker image and push it. Ex: push.docker.pilot
 push.$(1): DOCKER_TARGETS=$(1)
 push.$(1): docker.push
-- 
2.35.3

