From a7236baf4326d10cf35d50995f5df6908ee5d002 Mon Sep 17 00:00:00 2001
From: craigbox <craigbox@google.com>
Date: Wed, 23 Mar 2022 13:10:37 +0000
Subject: Update CODEOWNERS (#38086)

Why, indentation, why?
---
 CODEOWNERS | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CODEOWNERS b/CODEOWNERS
index 0fef37200e..2cc9994ed8 100644
--- a/CODEOWNERS
+++ b/CODEOWNERS
@@ -56,7 +56,7 @@ Makefile*                                                        @istio/wg-test-
 /tests/integration/security/                                     @istio/wg-security-maintainers
 /tests/integration/istioctl/                                     @istio/wg-user-experience-maintainers
 /tools/                                                          @istio/wg-test-and-release-maintainers
-/tools/packaging/common/envoy_bootstrap.json                  @istio/wg-networking-maintainers
+/tools/packaging/common/envoy_bootstrap.json                     @istio/wg-networking-maintainers
 /tools/istio-iptables/                                           @istio/wg-networking-maintainers
 /tools/istio-clean-iptables/                                     @istio/wg-networking-maintainers
 manifests/charts/global.yaml                                     @costinm @jacob-delgado @Monkeyanator
-- 
2.35.3

