From 5496a6e710a8e10419df203e4d5985ec1ef13c5a Mon Sep 17 00:00:00 2001
From: Eric Van Norman <ericvn@us.ibm.com>
Date: Thu, 24 Mar 2022 09:10:58 -0500
Subject: Make docker-copy Mac friendly and more execute flag correct (#38107)

---
 tools/docker-copy.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tools/docker-copy.sh b/tools/docker-copy.sh
index eb696395e9..6ba1d15a53 100755
--- a/tools/docker-copy.sh
+++ b/tools/docker-copy.sh
@@ -32,6 +32,7 @@ function may_copy_into_arch_named_sub_dir() {
   #   arm64/
   #   amd64/
   if [[ ${FILE_INFO} == *"ELF 64-bit LSB"* ]]; then
+    chmod 755 "${FILE}"
     case ${FILE_INFO} in
       *x86-64*)
         mkdir -p "${DOCKER_WORKING_DIR}/amd64/" && cp -rp "${FILE}" "${DOCKER_WORKING_DIR}/amd64/"
@@ -60,6 +61,7 @@ function may_copy_into_arch_named_sub_dir() {
     fi
 
   else
+    chmod 644 "${FILE}"
     cp -rp "${FILE}" "${DOCKER_WORKING_DIR}"
   fi
 }
@@ -68,5 +70,3 @@ mkdir -p "${DOCKER_WORKING_DIR}"
 for FILE in "${FILES[@]}"; do
   may_copy_into_arch_named_sub_dir "${FILE}"
 done
-find "${DOCKER_WORKING_DIR}" -type f -executable -exec chmod 755 {} \;
-find "${DOCKER_WORKING_DIR}" -type f ! -executable -exec chmod 644 {} \;
-- 
2.35.3

