From a21c1ca1f3848a28aff7dd88c9708954d43624b7 Mon Sep 17 00:00:00 2001
From: John Howard <howardjohn@google.com>
Date: Fri, 25 Mar 2022 13:50:31 -0700
Subject: Fix TODO for non-compression deb files (#38123)

---
 tools/packaging/packaging.mk | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/tools/packaging/packaging.mk b/tools/packaging/packaging.mk
index 85dc4a278b..fa42e6beb6 100644
--- a/tools/packaging/packaging.mk
+++ b/tools/packaging/packaging.mk
@@ -12,8 +12,7 @@ deb: ${TARGET_OUT_LINUX}/release/istio-sidecar.deb ${TARGET_OUT_LINUX}/release/i
 # fpm likes to add extremely high levels of compression. This is fine for release, but for local runs
 # where we are just pushing to a local registry (compressed again!), it adds ~1min to builds.
 ifneq ($(FAST_VM_BUILDS),)
-# TODO(debian): https://github.com/jordansissel/fpm/pull/1879
-DEB_COMPRESSION=
+DEB_COMPRESSION=--deb-compression=none
 RPM_COMPRESSION=--rpm-compression=none
 endif
 
-- 
2.35.3

