From 0814c7f28173a75d384209b3952967b5c41a7dd9 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Mon, 28 Mar 2022 13:01:13 -0700
Subject: Automator: update proxy@master in istio/istio@master (#38156)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index bbd47be8ac..62f3761ade 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "8bf9164ea9f26c3b38c0ad67f5247880c08e99d6"
+    "lastStableSHA": "979709d3518421dde1ea461f0b0e4d2cd5ab2420"
   }
 ]
-- 
2.35.3

