From 9252c480bcdf2eab9bd3fd72d9b4428c05b30277 Mon Sep 17 00:00:00 2001
From: Pengyuan Bian <bianpengyuan@google.com>
Date: Fri, 1 Apr 2022 15:19:27 -0700
Subject: Add comment about Wasm env clean up. (#38228)

---
 pkg/wasm/convert.go | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/pkg/wasm/convert.go b/pkg/wasm/convert.go
index 07ef023830..115a563334 100644
--- a/pkg/wasm/convert.go
+++ b/pkg/wasm/convert.go
@@ -132,6 +132,8 @@ func convert(resource *any.Any, cache Cache) (newExtensionConfig *any.Any, sendN
 			pullSecret = []byte(sec)
 		}
 		// Strip all internal env variables from VM env variable.
+		// These env variables are added by Istio control plane and meant to be consumed by the agent for image pulling control,
+		// thus should not be leaked to Envoy or the Wasm extension runtime.
 		delete(envs.KeyValues, model.WasmSecretEnv)
 		if len(envs.KeyValues) == 0 {
 			if len(envs.HostEnvKeys) == 0 {
-- 
2.35.3

