From 8bf4067477a0c1f8a3134c26ae60130f26df633f Mon Sep 17 00:00:00 2001
From: zirain <hejianpeng2@huawei.com>
Date: Wed, 6 Apr 2022 17:11:03 +0800
Subject: only get secretcontroller when needed (#38257)

---
 pilot/pkg/xds/ecds.go | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/pilot/pkg/xds/ecds.go b/pilot/pkg/xds/ecds.go
index 85bae9d773..4a2dd5a622 100644
--- a/pilot/pkg/xds/ecds.go
+++ b/pilot/pkg/xds/ecds.go
@@ -84,18 +84,18 @@ func (e *EcdsGenerator) Generate(proxy *model.Proxy, w *model.WatchedResource, r
 		}
 	}
 
-	// Generate the pull secrets first, which will be used when populating the extension config.
-	var secretController credscontroller.Controller
-	if e.secretController != nil {
-		var err error
-		secretController, err = e.secretController.ForCluster(proxy.Metadata.ClusterID)
-		if err != nil {
-			log.Warnf("proxy %s is from an unknown cluster, cannot retrieve certificates for Wasm image pull: %v", proxy.ID, err)
-			return nil, model.DefaultXdsLogDetails, nil
-		}
-	}
 	var secrets map[string][]byte
 	if len(secretResources) > 0 {
+		// Generate the pull secrets first, which will be used when populating the extension config.
+		var secretController credscontroller.Controller
+		if e.secretController != nil {
+			var err error
+			secretController, err = e.secretController.ForCluster(proxy.Metadata.ClusterID)
+			if err != nil {
+				log.Warnf("proxy %s is from an unknown cluster, cannot retrieve certificates for Wasm image pull: %v", proxy.ID, err)
+				return nil, model.DefaultXdsLogDetails, nil
+			}
+		}
 		// Inserts Wasm pull secrets in ECDS response, which will be used at xds proxy for image pull.
 		// Before forwarding to Envoy, xds proxy will remove the secret from ECDS response.
 		secrets = e.GeneratePullSecrets(proxy, updatedSecrets, secretResources, secretController, req)
-- 
2.35.3

