From 5955f705e20d29dd590cc11e1423e71db9dd822d Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Wed, 6 Apr 2022 21:59:14 -0700
Subject: Automator: update proxy@master in istio/istio@master (#38275)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 82d55fde34..9c2be1c063 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "af4ca235bf40fb11829f2f02cf678610641095a2"
+    "lastStableSHA": "77b805d60bcfef0e0c879fd3f145bae77c02f76d"
   }
 ]
-- 
2.35.3

