From 466c02050528263f22b80eabec8a19a5c62ea76e Mon Sep 17 00:00:00 2001
From: Eric Van Norman <ericvn@us.ibm.com>
Date: Fri, 8 Apr 2022 15:07:57 -0500
Subject: Fix Makefile related go-junit-report (#38302)

---
 Makefile.core.mk | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/Makefile.core.mk b/Makefile.core.mk
index c13929b429..f51e547b36 100644
--- a/Makefile.core.mk
+++ b/Makefile.core.mk
@@ -442,8 +442,16 @@ istioctl-install-container: istioctl
 
 .PHONY: test
 
+# This target sets JUNIT_REPORT to the location of the  go-junit-report binary.
+# This binary is provided in the build container. If it is not found, the build
+# container is not being used, so ask the user to install go-junit-report.
 JUNIT_REPORT := $(shell which go-junit-report 2> /dev/null || echo "${ISTIO_BIN}/go-junit-report")
 
+${ISTIO_BIN}/go-junit-report:
+	@echo "go-junit-report was not found in the build environment."
+	@echo "Please install go-junit-report (ex. go install github.com/jstemmer/go-junit-report@latest)"
+	@exit 1
+
 # This is just an alias for racetest now
 test: racetest ## Runs all unit tests
 
-- 
2.35.3

