From 95190cc57bf7a57a022cb4d6f9c787dcdb2d250e Mon Sep 17 00:00:00 2001
From: Eric Van Norman <ericvn@us.ibm.com>
Date: Tue, 12 Apr 2022 12:52:52 -0500
Subject: Update build-base-images.sh to use python3 (#38347)

---
 tools/build-base-images.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/build-base-images.sh b/tools/build-base-images.sh
index 0fa22b254b..dbc560e704 100755
--- a/tools/build-base-images.sh
+++ b/tools/build-base-images.sh
@@ -24,7 +24,7 @@ ROOT=$(dirname "$WD")
 set -ex
 
 toJson () {
-        python -c '
+        python3 -c '
 import sys, yaml, json
 yml = list(y for y in yaml.safe_load_all(sys.stdin) if y)
 if len(yml) == 1: yml = yml[0]
-- 
2.35.3

