From 4150520529c92dfc0c4e3ff32738b0108080c75f Mon Sep 17 00:00:00 2001
From: Rama Chavali <rama.rao@salesforce.com>
Date: Mon, 18 Apr 2022 18:50:13 +0530
Subject: add description for gateway config env var (#38423)

Signed-off-by: Rama Chavali <rama.rao@salesforce.com>
---
 pilot/pkg/features/pilot.go | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/pilot/pkg/features/pilot.go b/pilot/pkg/features/pilot.go
index c7c99bc172..83aec9f23f 100644
--- a/pilot/pkg/features/pilot.go
+++ b/pilot/pkg/features/pilot.go
@@ -81,7 +81,8 @@
 	// FilterGatewayClusterConfig controls if a subset of clusters(only those required) should be pushed to gateways
 	// TODO enable by default once https://github.com/istio/istio/issues/28315 is resolved
 	// Currently this may cause a bug when we go from N clusters -> 0 clusters -> N clusters
-	FilterGatewayClusterConfig = env.RegisterBoolVar("PILOT_FILTER_GATEWAY_CLUSTER_CONFIG", false, "").Get()
+	FilterGatewayClusterConfig = env.RegisterBoolVar("PILOT_FILTER_GATEWAY_CLUSTER_CONFIG", false,
+		"If enabled, Pilot will send only clusters that referenced in gateway virtual services attached to gateway").Get()
 
 	DebounceAfter = env.RegisterDurationVar(
 		"PILOT_DEBOUNCE_AFTER",
-- 
2.35.3

