From 5645dec7fbb67e3ef0bb68a0a074fdbd0b6b98d0 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Mon, 18 Apr 2022 16:19:39 -0700
Subject: Automator: update proxy@master in istio/istio@master (#38439)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index d82c197183..de8fe12943 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "af156453fd0af68270a584b560d5f711ac0b5812"
+    "lastStableSHA": "7a9bfa6a012c997c068a11eaa0e7a00cb4b788c1"
   }
 ]
-- 
2.35.3

