From c4155536923945460dc9c1f22f82478c49a256f6 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Tue, 19 Apr 2022 16:38:08 -0700
Subject: Automator: update proxy@master in istio/istio@master (#38456)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index de8fe12943..0cfe07891b 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "7a9bfa6a012c997c068a11eaa0e7a00cb4b788c1"
+    "lastStableSHA": "9bf954c37457e9e2f800c9220fea44f5b5e456f4"
   }
 ]
-- 
2.35.3

