From 7c583c4600decefc6bf7f8216dcd65edb0ed095a Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Wed, 20 Apr 2022 16:24:46 -0700
Subject: Automator: update proxy@master in istio/istio@master (#38481)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 0cfe07891b..3a0e585ee5 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "9bf954c37457e9e2f800c9220fea44f5b5e456f4"
+    "lastStableSHA": "15985747c09e4ec385321094f35bf71b35b1cb4e"
   }
 ]
-- 
2.35.3

