From 2a98015b0997fa18e77ee598c5f468691fc81691 Mon Sep 17 00:00:00 2001
From: Eric Van Norman <ericvn@us.ibm.com>
Date: Thu, 21 Apr 2022 09:54:13 -0500
Subject: Missing space (#38498)

---
 prow/lib.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/prow/lib.sh b/prow/lib.sh
index c373e652c4..5b4bc12d79 100755
--- a/prow/lib.sh
+++ b/prow/lib.sh
@@ -114,7 +114,7 @@ function build_images() {
   targets="docker.pilot docker.proxyv2 "
 
   # use ubuntu:jammy to test vms by default
-  nonDistrolessTargets="docker.app docker.app_sidecar_ubuntu_jammy docker.ext-authz"
+  nonDistrolessTargets="docker.app docker.app_sidecar_ubuntu_jammy docker.ext-authz "
   if [[ "${JOB_TYPE:-presubmit}" == "postsubmit" ]]; then
     # We run tests across all VM types only in postsubmit
     nonDistrolessTargets+="docker.app_sidecar_ubuntu_xenial docker.app_sidecar_debian_11  docker.app_sidecar_centos_7 "
-- 
2.35.3

