From 74eafdedc0dd36c988e532a04d3ad242020a9b92 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Thu, 21 Apr 2022 16:37:10 -0700
Subject: Automator: update proxy@master in istio/istio@master (#38510)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 3a0e585ee5..23d37b2ac5 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "15985747c09e4ec385321094f35bf71b35b1cb4e"
+    "lastStableSHA": "ab1dc1cad808ff8d0a1f98c8a27723fab9e4f28c"
   }
 ]
-- 
2.35.3

