From 8467b5a920e70b91b7c6d5747c867c3fa15e6122 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Sat, 23 Apr 2022 10:44:51 -0700
Subject: Automator: update proxy@master in istio/istio@master (#38538)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 23d37b2ac5..4474d736ad 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "ab1dc1cad808ff8d0a1f98c8a27723fab9e4f28c"
+    "lastStableSHA": "bbce06b576b5a611583ccfc60388a28a5243bf2d"
   }
 ]
-- 
2.35.3

