From 527a5135eaed861cb6ee4e68c5a3bab2dc34fe72 Mon Sep 17 00:00:00 2001
From: Eric Van Norman <ericvn@us.ibm.com>
Date: Mon, 25 Apr 2022 15:09:05 -0500
Subject: Manual revert of #38344 (#38563)

---
 docker/Dockerfile.distroless | 23 ++++++++++++++++++++---
 1 file changed, 20 insertions(+), 3 deletions(-)

diff --git a/docker/Dockerfile.distroless b/docker/Dockerfile.distroless
index 8c5f3e4c13..fe94784bf1 100644
--- a/docker/Dockerfile.distroless
+++ b/docker/Dockerfile.distroless
@@ -1,3 +1,20 @@
-# istio-static image is prebuilt with 1337:1337.
-# https://github.com/istio/distroless/tree/iptables/istio
-FROM gcr.io/istio-release/distroless/istio-static@sha256:d6fa9db9548b5772860fecddb11d84f9ebd7e0321c0cb3c02870402680cc315f as distroless_source
\ No newline at end of file
+# prepare a distroless source context to copy files from
+FROM gcr.io/distroless/static-debian11@sha256:d6fa9db9548b5772860fecddb11d84f9ebd7e0321c0cb3c02870402680cc315f as distroless_source
+
+# prepare a base dev to modify file contents
+FROM ubuntu:focal as ubuntu_source
+
+# Modify contents of container
+COPY --from=distroless_source /etc/ /home/etc
+COPY --from=distroless_source /home/nonroot /home/nonroot
+RUN echo istio-proxy:x:1337: >> /home/etc/group
+RUN echo istio-proxy:x:1337:1337:istio-proxy:/nonexistent:/sbin/nologin >> /home/etc/passwd
+
+# Customize distroless with the following:
+# - password file
+# - groups file
+# - /home/nonroot directory
+FROM distroless_source
+COPY --from=ubuntu_source /home/etc/passwd /etc/passwd
+COPY --from=ubuntu_source /home/etc/group /etc/group
+COPY --from=ubuntu_source /home/nonroot /home/nonroot
\ No newline at end of file
-- 
2.35.3

