From 5bc0c20d7a1be4633c40a4d7929d04c2cd58c43f Mon Sep 17 00:00:00 2001
From: sergii-ssh <83605538+sergii-ssh@users.noreply.github.com>
Date: Mon, 25 Apr 2022 18:53:04 -0400
Subject: Switch Istio base image to use Ubuntu Jammy (#38577)

---
 docker/Dockerfile.base | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docker/Dockerfile.base b/docker/Dockerfile.base
index d978ccf46a..847e22b291 100644
--- a/docker/Dockerfile.base
+++ b/docker/Dockerfile.base
@@ -1,4 +1,4 @@
-FROM ubuntu:focal
+FROM ubuntu:jammy
 
 ENV DEBIAN_FRONTEND=noninteractive
 
-- 
2.35.3

