From 143d2f140bf6b04147ff4a910d617ea3bea5523d Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Mon, 25 Apr 2022 18:25:04 -0700
Subject: Automator: update proxy@master in istio/istio@master (#38582)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 4474d736ad..971598a2cc 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "bbce06b576b5a611583ccfc60388a28a5243bf2d"
+    "lastStableSHA": "b6b3e1134ddca503ec33a02135fb525aec5c98d5"
   }
 ]
-- 
2.35.3

