From 34577f0455e6a4c55ac4a46d0458a2302a35f8d1 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Tue, 26 Apr 2022 18:11:00 -0700
Subject: Automator: update proxy@master in istio/istio@master (#38606)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 971598a2cc..b652a19748 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "b6b3e1134ddca503ec33a02135fb525aec5c98d5"
+    "lastStableSHA": "9c98ad4bd3ceb310511562e5cd26f1d0514cb5fa"
   }
 ]
-- 
2.35.3

