From 57dcbfe00ad07e8ad4eba387aaa603af7a5a348b Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Wed, 27 Apr 2022 18:36:33 -0700
Subject: Automator: update proxy@master in istio/istio@master (#38631)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index b652a19748..4acffda7db 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "9c98ad4bd3ceb310511562e5cd26f1d0514cb5fa"
+    "lastStableSHA": "771bdb3e930f2cdacefbb1fc8c0634a0b63afc00"
   }
 ]
-- 
2.35.3

