From e67c34b8c005ec13d2a3b2dd897b45124a02a5ca Mon Sep 17 00:00:00 2001
From: zirain <hejianpeng2@huawei.com>
Date: Sat, 30 Apr 2022 16:59:34 +0800
Subject: remove comment (#38668)

---
 istioctl/pkg/multicluster/options.go | 1 -
 1 file changed, 1 deletion(-)

diff --git a/istioctl/pkg/multicluster/options.go b/istioctl/pkg/multicluster/options.go
index d28e201fb6..c03d36e3eb 100644
--- a/istioctl/pkg/multicluster/options.go
+++ b/istioctl/pkg/multicluster/options.go
@@ -21,7 +21,6 @@
 	"k8s.io/client-go/tools/clientcmd"
 )
 
-// TODO(ayj) - add to istio.io/api/annotations
 const clusterNameAnnotationKey = "networking.istio.io/cluster"
 
 // KubeOptions contains kubernetes options common to all commands.
-- 
2.35.3

