From dcb4692df60088ccb371879c12b243ca95a3f717 Mon Sep 17 00:00:00 2001
From: zirain <hejianpeng2@huawei.com>
Date: Tue, 3 May 2022 21:59:35 +0800
Subject: make-addons (#38550)

---
 Makefile.core.mk | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Makefile.core.mk b/Makefile.core.mk
index d11d97d205..75a28179c5 100644
--- a/Makefile.core.mk
+++ b/Makefile.core.mk
@@ -333,6 +333,9 @@ update-golden: refresh-goldens
 gen-charts:
 	@echo "This target is no longer required and will be removed in the future"
 
+gen-addons:
+	manifests/addons/gen.sh
+
 gen: \
 	mod-download-go \
 	go-gen \
@@ -342,6 +345,7 @@ gen: \
 	proto \
 	copy-templates \
 	gen-kustomize \
+	gen-addons \
 	update-golden ## Update all generated code.
 
 gen-check: gen check-clean-repo
-- 
2.35.3

