From db0c1f49b8af070ee02e5e6008372770d614bea3 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Tue, 3 May 2022 13:42:35 -0700
Subject: Automator: update common-files@master in istio/istio@master (#38705)

---
 .gitattributes          | 1 +
 common/.commonfiles.sha | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/.gitattributes b/.gitattributes
index 297d227c81..eea5a1ec86 100644
--- a/.gitattributes
+++ b/.gitattributes
@@ -8,3 +8,4 @@
 *.gen.yaml linguist-generated=true
 *.gen.json linguist-generated=true
 *_pb2.py linguist-generated=true
+go.sum merge=union
diff --git a/common/.commonfiles.sha b/common/.commonfiles.sha
index e41057a756..8280a87979 100644
--- a/common/.commonfiles.sha
+++ b/common/.commonfiles.sha
@@ -1 +1 @@
-78d4a03042d695220b4ade5c17f071898d466b58
+d40d1b75c481bb08a4e5de631c26a4352d7039fd
-- 
2.35.3

