From c13b7924eeb59cb59186268cede82786e378d29a Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Thu, 5 May 2022 12:47:01 -0700
Subject: Update BASE_VERSION to master-2022-05-05T19-01-00 (#38757)

---
 Makefile.core.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.core.mk b/Makefile.core.mk
index 75a28179c5..bd8c9b13e3 100644
--- a/Makefile.core.mk
+++ b/Makefile.core.mk
@@ -22,7 +22,7 @@ SHELL := /bin/bash -o pipefail
 export VERSION ?= 1.15-dev
 
 # Base version of Istio image to use
-BASE_VERSION ?= master-2022-04-29T19-00-55
+BASE_VERSION ?= master-2022-05-05T19-01-00
 
 export GO111MODULE ?= on
 export GOPROXY ?= https://proxy.golang.org
-- 
2.35.3

