From a71225e9666764f0cb0a030c10ec3b245db68dd8 Mon Sep 17 00:00:00 2001
From: Zhonghu Xu <xuzhonghu@huawei.com>
Date: Tue, 10 May 2022 00:28:52 +0800
Subject: Make use protobuf codec for k8s api resources (#38658)

---
 pkg/kube/client.go | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/pkg/kube/client.go b/pkg/kube/client.go
index cfa50d0528..91b0889c91 100644
--- a/pkg/kube/client.go
+++ b/pkg/kube/client.go
@@ -374,6 +374,8 @@ func newClientInternal(clientFactory util.Factory, revision string) (*client, er
 		return nil, err
 	}
 
+	config := rest.CopyConfig(c.config)
+	config.ContentType = runtime.ContentTypeProtobuf
 	c.Interface, err = kubernetes.NewForConfig(c.config)
 	c.kube = c.Interface
 	if err != nil {
-- 
2.35.3

