From 32578c4a999629dea95946be2cc90b5fe037564f Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Wed, 18 May 2022 13:08:41 -0700
Subject: Update BASE_VERSION to master-2022-05-18T19-02-54 (#39008)

---
 Makefile.core.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.core.mk b/Makefile.core.mk
index 706cfe9085..3c25036b89 100644
--- a/Makefile.core.mk
+++ b/Makefile.core.mk
@@ -22,7 +22,7 @@ SHELL := /bin/bash -o pipefail
 export VERSION ?= 1.15-dev
 
 # Base version of Istio image to use
-BASE_VERSION ?= master-2022-05-16T21-51-27
+BASE_VERSION ?= master-2022-05-18T19-02-54
 
 export GO111MODULE ?= on
 export GOPROXY ?= https://proxy.golang.org
-- 
2.35.3

