From b45a7281523086d47b7db84615e0515a533a10a1 Mon Sep 17 00:00:00 2001
From: Zhonghu Xu <xuzhonghu@huawei.com>
Date: Wed, 25 May 2022 08:22:40 +0800
Subject: Add myself (#39101)

---
 CODEOWNERS | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CODEOWNERS b/CODEOWNERS
index 2cc9994ed8..d2afe3bdc3 100644
--- a/CODEOWNERS
+++ b/CODEOWNERS
@@ -1,4 +1,4 @@
-*                                                                @costinm @linsun @howardjohn @nmittler @ericvn
+*                                                                @costinm @linsun @howardjohn @nmittler @ericvn @hzxuzhonghu
 /.gitattributes                                                  @istio/wg-test-and-release-maintainers
 /.gitignore                                                      @istio/wg-test-and-release-maintainers
 /Makefile.core.mk                                                @istio/wg-test-and-release-maintainers
-- 
2.35.3

