From dde2015562d8bf0216237322a15dd4625b3530c2 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Wed, 25 May 2022 10:58:40 -0700
Subject: Automator: update proxy@master in istio/istio@master (#39132)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index ac8787278c..6222a7af45 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "44e4301ccecc331d1d0682fffc3f3462a45f0ea3"
+    "lastStableSHA": "6946853ea44fc4d63a8a7e89e3040c1d9889112d"
   }
 ]
-- 
2.35.3

