From 201171bdd5e1d5978e714b08dea91d363d6af21c Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Mon, 14 Feb 2022 08:29:37 -0800
Subject: Update BASE_VERSION to 1.12-2022-02-11T19-01-16 (#37299)

---
 Makefile.core.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.core.mk b/Makefile.core.mk
index 5379d15d25..bfd34bcd05 100644
--- a/Makefile.core.mk
+++ b/Makefile.core.mk
@@ -22,7 +22,7 @@ SHELL := /bin/bash -o pipefail
 export VERSION ?= 1.12-dev
 
 # Base version of Istio image to use
-BASE_VERSION ?= 1.12-2022-01-17T19-01-33
+BASE_VERSION ?= 1.12-2022-02-11T19-01-16
 
 export GO111MODULE ?= on
 export GOPROXY ?= https://proxy.golang.org
-- 
2.35.3

