From 28a3b6b7fa16c3f552e521953542bfa021e3a27a Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Thu, 17 Feb 2022 16:32:05 -0800
Subject: Automator: update proxy@release-1.12 in istio/istio@release-1.12
 (#37419)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index cdaf41fa21..cc5e88d67a 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "ba0bf13e6328b233e1249505f181f9ab30898529"
+    "lastStableSHA": "dd4c533b72e9dccf8dcb410811b34a8de9ed3913"
   }
 ]
-- 
2.35.3

