From 609d79e7ebfc0e6505fa593276aae3aa6d9dccac Mon Sep 17 00:00:00 2001
From: Daniel Grimm <dgrimm@redhat.com>
Date: Wed, 4 May 2022 18:14:40 +0200
Subject: OSSM-1460 Simplify MaistraDiscoveryNamespacesFilter

---
 .../kube/controller/filter/discoverynamespaces.go            | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/pilot/pkg/serviceregistry/kube/controller/filter/discoverynamespaces.go b/pilot/pkg/serviceregistry/kube/controller/filter/discoverynamespaces.go
index e13fe5b221..c898305bf7 100644
--- a/pilot/pkg/serviceregistry/kube/controller/filter/discoverynamespaces.go
+++ b/pilot/pkg/serviceregistry/kube/controller/filter/discoverynamespaces.go
@@ -236,9 +236,8 @@ func (d *maistraDiscoveryNamespacesFilter) SetNamespaces(namespaces ...string) {
 }
 
 func (d *maistraDiscoveryNamespacesFilter) Filter(obj interface{}) bool {
-	d.lock.RLock()
-	defer d.lock.RUnlock()
-	return d.namespaces.Has(obj.(metav1.Object).GetNamespace())
+	// xns-informers make sure we only receive objects in member namespaces
+	return true
 }
 
 func (d *maistraDiscoveryNamespacesFilter) SelectorsChanged(
-- 
2.35.3

