From 53ab7dcafe6d6cf2fb9f09c852df1934b7be9b71 Mon Sep 17 00:00:00 2001
From: Greg Hanson <gregory.hanson@solo.io>
Date: Thu, 14 Apr 2022 12:47:06 -0400
Subject: manual cherry-pick 38211 (#38291)

---
 pilot/docker/Dockerfile.proxyv2 | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pilot/docker/Dockerfile.proxyv2 b/pilot/docker/Dockerfile.proxyv2
index 02cded141b..1cdc296ae8 100644
--- a/pilot/docker/Dockerfile.proxyv2
+++ b/pilot/docker/Dockerfile.proxyv2
@@ -11,8 +11,8 @@ FROM gcr.io/istio-release/base:${BASE_VERSION} as debug
 # This image is a custom built debian11 distroless image with multiarchitecture support.
 # It is built on the base distroless image, with iptables binary and libraries added
 # The source can be found at https://github.com/istio/distroless/tree/iptables
-# This version is from commit 20b20ec36f4621830ff0bfdec519577074df47ca.
-FROM gcr.io/istio-release/iptables@sha256:bae9287d64be13179b7bc794ec3db26bd5c5fe3fb591c484992366314c9a7d3d as distroless
+# This version is from commit 11fea21b647493aa1ce54db1b858bd3e32a6edca.
+FROM gcr.io/istio-release/iptables@sha256:a5d55f7cd308eeb1c0f58271bfbfec260c922d16a56ad34e3a6b5f1bf8ef80a2 as distroless
 
 # This will build the final image based on either debug or distroless from above
 # hadolint ignore=DL3006
-- 
2.35.3

