From f75019c8a96002b96d0abeb9eb464b7b467ebff7 Mon Sep 17 00:00:00 2001
From: Eric Van Norman <ericvn@us.ibm.com>
Date: Fri, 29 Oct 2021 11:14:10 -0500
Subject: Fixup UPDATE_BRANCH (#35798)

---
 bin/update_deps.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bin/update_deps.sh b/bin/update_deps.sh
index b32317e035..135e99442f 100755
--- a/bin/update_deps.sh
+++ b/bin/update_deps.sh
@@ -16,7 +16,7 @@
 
 set -e
 
-UPDATE_BRANCH=${UPDATE_BRANCH:-"master"}
+UPDATE_BRANCH=${UPDATE_BRANCH:-"release-1.12"}
 
 SCRIPTPATH="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 ROOTDIR=$(dirname "${SCRIPTPATH}")
-- 
2.35.3

