From 860908606ea391af0a6156a44645183a03a41f87 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Wed, 26 Jan 2022 06:30:28 -0800
Subject: Automator: update proxy@release-1.12 in istio/istio@release-1.12
 (#37012)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 9f827fbe22..cdaf41fa21 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "7f9b7ffd99c39e07657e919459e00b68604223bc"
+    "lastStableSHA": "ba0bf13e6328b233e1249505f181f9ab30898529"
   }
 ]
-- 
2.31.1

