From ea1eab82afc75bfeda993cce7110b03ddeca3415 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Wed, 3 Nov 2021 10:24:09 -0700
Subject: Automator: update proxy@release-1.12 in istio/istio@release-1.12
 (#35875)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 1dd320c055..06f2829598 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "68917cdb85bc849657b6c77b7dca238dbffa9ee4"
+    "lastStableSHA": "2ed402c748961606172482d48065f9e4a122259a"
   }
 ]
-- 
2.35.3

