From 9ceeb554e74fa41d530c53a11dd2c83eeb6607b4 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Fri, 18 Feb 2022 16:36:28 -0800
Subject: Automator: update proxy@release-1.12 in istio/istio@release-1.12
 (#37446)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index cc5e88d67a..7fa085c352 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "dd4c533b72e9dccf8dcb410811b34a8de9ed3913"
+    "lastStableSHA": "e2bdb6e755aec2e4b20dfbc8d3740bf496b356e7"
   }
 ]
-- 
2.31.1

