From 5663441164334aa2447fecab8682b5285a8eee30 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Wed, 23 Feb 2022 20:21:06 -0800
Subject: Automator: update proxy@release-1.12 in istio/istio@release-1.12
 (#37458)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index 7fa085c352..55d1a30d07 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "e2bdb6e755aec2e4b20dfbc8d3740bf496b356e7"
+    "lastStableSHA": "4b277b4891f712dc9395533014aa41367dd2b658"
   }
 ]
-- 
2.31.1

