From 349b8929ad345222928587427d0cfc5f0fe93c55 Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Thu, 24 Feb 2022 14:33:35 -0800
Subject: Update BASE_VERSION to 1.12-2022-02-23T19-01-16 (#37494)

---
 Makefile.core.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.core.mk b/Makefile.core.mk
index bfd34bcd05..eb4b53f89f 100644
--- a/Makefile.core.mk
+++ b/Makefile.core.mk
@@ -22,7 +22,7 @@ SHELL := /bin/bash -o pipefail
 export VERSION ?= 1.12-dev
 
 # Base version of Istio image to use
-BASE_VERSION ?= 1.12-2022-02-11T19-01-16
+BASE_VERSION ?= 1.12-2022-02-23T19-01-16
 
 export GO111MODULE ?= on
 export GOPROXY ?= https://proxy.golang.org
-- 
2.31.1

