From 7fb2c2f9145f5f4273fecf92623636bf4f8cd52c Mon Sep 17 00:00:00 2001
From: Jonh Wendell <jonh.wendell@redhat.com>
Date: Tue, 8 Feb 2022 16:09:10 -0500
Subject: Only use gcloud if we have credentials

---
 prow/lib.sh | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/prow/lib.sh b/prow/lib.sh
index be43b174f6..ed6a925040 100755
--- a/prow/lib.sh
+++ b/prow/lib.sh
@@ -41,6 +41,10 @@ function trace() {
 }
 
 function setup_gcloud_credentials() {
+  if [[ -z "${GOOGLE_APPLICATION_CREDENTIALS:-}" ]]; then
+    return
+  fi
+
   if [[ $(command -v gcloud) ]]; then
     gcloud auth configure-docker -q
   elif [[ $(command -v docker-credential-gcr) ]]; then
-- 
2.31.1

