From 77fb11bea4985c98abcc9edc7c6facdcd22ac9ae Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Wed, 16 Mar 2022 21:29:51 -0700
Subject: Automator: update proxy@release-1.12 in istio/istio@release-1.12
 (#37978)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index f296da58e6..c2fd577e0c 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "312cad6e24a1d009bca4111166083af569412ba0"
+    "lastStableSHA": "af4302eb72af1ca578be5f41221214ba5c61d1f3"
   }
 ]
-- 
2.35.3

