From ad05d6bfb5977fdf0fc51c440e5ecd5ae837d82d Mon Sep 17 00:00:00 2001
From: Istio Automation <istio-testing-bot@google.com>
Date: Thu, 11 Nov 2021 16:08:45 -0800
Subject: Automator: update proxy@release-1.12 in istio/istio@release-1.12
 (#36039)

---
 istio.deps | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/istio.deps b/istio.deps
index a8a4bdf7dd..2053eca502 100644
--- a/istio.deps
+++ b/istio.deps
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "765481d0d5b34baa503eceeceffd6e7cfddd08f3"
+    "lastStableSHA": "1e1f977c7964e3d95f706ed10148e2bcb4aace33"
   }
 ]
-- 
2.35.3

