From a0c7a3355331ef20354e3e1682dc13b7e6bcf4c1 Mon Sep 17 00:00:00 2001
From: Karol Szwaj <karol.szwaj@gmail.com>
Date: Mon, 28 Mar 2022 21:41:12 +0200
Subject: Remove double generation of IstioOperatorSpec in istioctl upgrade
 (#38109)

---
 operator/cmd/mesh/upgrade.go | 11 +++--------
 1 file changed, 3 insertions(+), 8 deletions(-)

diff --git a/operator/cmd/mesh/upgrade.go b/operator/cmd/mesh/upgrade.go
index e30f2480d0..9707adf523 100644
--- a/operator/cmd/mesh/upgrade.go
+++ b/operator/cmd/mesh/upgrade.go
@@ -219,24 +219,19 @@ func upgrade(rootArgs *rootArgs, args *upgradeArgs, l clog.Logger) (err error) {
 
 	waitForConfirmation(args.skipConfirmation || rootArgs.dryRun, l)
 
-	_, iop, err := manifest.GenerateConfig(args.inFilenames, setFlags, args.force, restConfig, l)
-	if err != nil {
-		return fmt.Errorf("generate config: %v", err)
-	}
-
 	extendedClient, err := kube.NewExtendedClient(kube.BuildClientCmd(args.kubeConfigPath, args.context), opts.Revision)
 	if err != nil {
 		return fmt.Errorf("create Kubernetes client: %v", err)
 	}
 	// Detect whether previous installation exists prior to performing the installation.
 	exists := revtag.PreviousInstallExists(context.Background(), extendedClient)
-	pilotEnabled := iop.Spec.Components.Pilot != nil && iop.Spec.Components.Pilot.Enabled.Value
-	rev := iop.Spec.Revision
+	pilotEnabled := targetIOP.Spec.Components.Pilot != nil && targetIOP.Spec.Components.Pilot.Enabled.Value
+	rev := targetIOP.Spec.Revision
 	if rev == "" && pilotEnabled {
 		_ = revtag.DeleteTagWebhooks(context.Background(), extendedClient, revtag.DefaultRevisionName)
 	}
 	// Apply the Istio Control Plane specs reading from inFilenames to the cluster
-	iop, err = InstallManifests(targetIOP, args.force, rootArgs.dryRun, restConfig, client, args.readinessTimeout, l)
+	iop, err := InstallManifests(targetIOP, args.force, rootArgs.dryRun, restConfig, client, args.readinessTimeout, l)
 	if err != nil {
 		return fmt.Errorf("failed to apply the Istio Control Plane specs. Error: %v", err)
 	}
-- 
2.35.3

